.github/workflows/test.yml

```diff
@@ -50,4 +50,4 @@ jobs:
         GCP_REGION: ${{ secrets.GCP_REGION }}
         GCP_PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
       run: |
-        pytest --verbose --nbval-lax python-recipes/RAG/ python-recipes/vector-search python-recipes/redis-intro --ignore python-recipes/RAG/05_nvidia_ai_rag_redis.ipynb
\ No newline at end of file
+        pytest --verbose --nbval-lax python-recipes/RAG/ python-recipes/vector-search python-recipes/redis-intro --ignore python-recipes/RAG/05_nvidia_ai_rag_redis.ipynb python-recipes/semantic-cache/doc2cache_llama3_1.ipynb
```

Comment:
Seems like you might need to double up on the --ignore flag (which is dumb) cause it still looks like it's trying to run tests for this

<img width="1089" alt="image" src="https://github.com/user-attachments/assets/336eb839-d215-4de0-adfe-d6c8d68545d2">

Reply:
will fix